Automattic/jetpack
# Jetpack Sync: Add 'woocommerce_analytics_first_activation' in options' whitelist

Adds `woocommerce_analytics_first_activation` in Sync options' whitelist

WPCOM couterpart: D163191-code

Related to: https://github.com/woocommerce/woocommerce-analytics/pull/180

## Proposed changes:

* `Automattic\Jetpack\Sync\Defailts`: Add `woocommerce_analytics_first_activation` in `$default_options_whitelist`

### Other information:


## Jetpack product discussion


pf5801-16d-p2#comment-566

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* In `cli` update the corresponding option via `update_option('woocommerce_analytics_first_activation', true)` and ensure its value is updated in the Cache site
